frontend/pages/admin/IntegrationsPage/cards/AutomaticEnrollment/WindowsAutomaticEnrollmentPage/WindowsAutomaticEnrollmentPage.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { useContext } from "react";
+
+import PATHS from "router/paths";
+import { AppContext } from "context/app";
+
+// @ts-ignore
+import InputField from "components/forms/fields/InputField";
+import BackLink from "components/BackLink";
+import MainContent from "components/MainContent";
+import CustomLink from "components/CustomLink/CustomLink";
+
+const generateMdmTermsOfUseUrl = (domain: string) => {
+  return `${domain}/api/v1/fleet/mdm/microsoft/terms_of_use`;
+};
+
+const generateMdmDiscoveryUrl = (domain: string) => {
+  return `${domain}/api/v1/fleet/mdm/microsoft/discovery`;
+};
```

Comment:
At first glance, these seemed like they should be coming from `endpoints`. But based on the current usage, I think you made the right call to keep them with just this page.